### PR TITLE
Finish Capstone Project 01

### PR DESCRIPTION
**Pull request by peter-akhigbe**
## Branch Changes

- Set up the repository and tools.
- Create the Home and About pages for mobile.
- Adapt the 2 pages to desktop.
- Implement the section "Featured speakers" with dynamic HTML.

### CODE REVIEW

**Comment by wayungi** [comment]
@peter-akhigbe , please fix your live demo link

**Comment by peter-akhigbe** [comment]
Because there's no index.html file in the main branch, the live link cannot work until I merge the feature branch.

**Comment by wayungi** [comment]
please do

**Comment by wayungi** [comment]
please test the links of your live demo

**Comment by peter-akhigbe** [comment]
I have fixed the issue with broken links

File: index.html [hunk 1]
```diff
@@ -0,0 +1,189 @@
...
+          <p class="p">
+            Get opportunities to network with CC affiliates around the world,
+            also after the summit.
+          </p>
+        </div>
+      </div>
+
+      <div class="red-box">Join CC Global Summit 2015</div>
+      <div class="whole-program">SEE THE WHOLE PROGRAM</div>
+    </section>
```
Comment (wayungi) [inline index.html:146]: -  [ ] Very good work implementing the main **program section**, please place the articles correctly as in the design.

**design**
ms1
**Your implementation**
ms2

- [ ] What do you think about using the `<article>` tag for each of the items in the main programme 🤔 ?

File: index.html [hunk 2]
```diff
@@ -0,0 +1,189 @@
...
+
+    <section id="sponsor" class="section-3">
+      <div class="h2-red-line">
+        <h2 class="h2">Featured Speakers</h2>
+        <hr class="red-line" />
+      </div>
+
+      <div class="container">
+        <!-- dynamically generating speakers here -->
+      </div>
```
Comment (wayungi) [inline index.html:158]: - [ ] You have done a very good job generating this content dynamically. You are left with fine-tuning the styles to match the original design. please look at the discrepancies below. Your images are not aligned at the top with the text on the left

**Design**
align1

**Your implementation**
align2

**--------mobile design------------**
 - [ ] Please look at the mobile designs below.  In the middle section, individual articles have their elements organized in a horizontal row, with the image at the **left**, the title in the center, and the description at the right.
 
 **Design**
mob-ali-1

**Your design**
mob-al2

File: About/about.html [hunk 3]
```diff
@@ -0,0 +1,163 @@
...
+              </li>
+            </ul>
+          </div>
+        </nav>
+      </header>
+
+      <!-- Section one -->
+      <section class="section-1">
+        <div class="hello">"Hello! Sharing world"</div>
+        <h1 class="h1">Creative Commons Global Submit 2015</h1>
```
Comment (wayungi) [inline About/about.html]: - [ OPTIONAL] Giving class names that are HTML tags is not semantic. It cause cause bugs later. What do you think about changing this ? 💭

File: main.js [hunk 4]
```diff
@@ -0,0 +1,129 @@
...
+    image: 'images/speakers/Margaret_Hamilton.jpg',
+    name: 'Margaret Hamilton',
+    headline: 'CEO of Creative Commons, ex COO of the Mozilla Foundation',
+    p: 'Ryan had been leading open-source projects at the Mozilla Foundation such as the open source movement.',
+  },
+];
+
+// Dynamically Generating Spekaers HTML
+speakers.forEach((speaker) => {
+  const innerContainer = document.createElement('div');
```
Comment (wayungi) [inline main.js:57]: - [ OPTIONAL] Following semantic writing, `const innerContainer = document.createElement('div');` line should create an `article` not a div. Whats your take ?

**Review by wayungi — CHANGES_REQUESTED** [review]
Hi @peter-akhigbe  !!

image

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights
- [ ] Profesional README ✔️ 
- [ ] All linter checks passing ✔️ 
- [ ] Good commit messages ✔
- [ ] No merge conflicts ✔️ 
- [ ]  Correct GitHub flow ✔️ 
- [ ] Descriptive PR body

## Required Changes ♻️
_Check the comments under the review._
- [ ] Please add a video presentation link of your project in the README 
## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: about.html [hunk 5]
```diff
@@ -0,0 +1,173 @@
...
+          competition from 8. June to 7. July.
+          <br />
+          After online public voting and Selection Committee's judging process,
+          the logo from Naresh Agrawal won the prize. After consultation with
+          Mr. Agrawal we optimized the prize-winning-logo for this year's summit
+          as following:
+        </p>
+      </div>
+
+      <img class="logo" src="images/cc-global.png" alt="image" />
```
Comment (Abdessittir) [inline about.html:129]: - [OPTIONAL] I highly recommend that you put a more descriptive `alt` to tell what this image is about, what do you think?

File: about.html [hunk 6]
```diff
@@ -0,0 +1,173 @@
...
+        <div class="event event-2">
+          <h3 class="h3">2011</h3>
+          <p class="p">CC Global Summit 2011 in Warsaw</p>
+        </div>
+      </div>
+    </section>
+
+    <!-- Footer Section -->
+    <footer class="footer">
+      <img class="logo" src="images/logo.png" alt="image" />
```
Comment (Abdessittir) [inline about.html:155]: - [OPTIONAL] I highly recommend that you put a more descriptive `alt` to tell what this image is about, what do you think?

File: index.html [hunk 7]
```diff
@@ -0,0 +1,198 @@
...
+    <section id="program" class="section-2">
+      <div class="h2-red-line">
+        <h2 class="h2">Main Program</h2>
+        <hr class="red-line" />
+      </div>
+
+      <div class="container">
+        <!-- Box 1 -->
+        <div class="inner-container">
+          <img class="icon" src="images/program/lecture.png" alt="image" />
```
Comment (Abdessittir) [inline index.html:95]: - [OPTIONAL] I highly recommend that you put a more descriptive `alt` to tell what this image is about, what do you think?

File: index.html [hunk 8]
```diff
@@ -0,0 +1,198 @@
...
+          <h3 class="h3">Lecture</h3>
+          <p class="p">
+            Listen to the speakers from countries about the messages of sharing
+            and opening.
+          </p>
+        </div>
+
+        <!-- Box 2 -->
+        <div class="inner-container">
+          <img class="icon" src="images/program/exhibition.png" alt="image" />
```
Comment (Abdessittir) [inline index.html:105]: - [OPTIONAL] I highly recommend that you put a more descriptive `alt` to tell what this image is about, what do you think?

File: index.html [hunk 9]
```diff
@@ -0,0 +1,198 @@
...
+          <h3 class="h3">CC Exibition</h3>
+          <p class="p">
+            Appreciate various creations applying CC license of artists,
+            organized from Art Center Nabi.
+          </p>
+        </div>
+
+        <!-- Box 3 -->
+        <div class="inner-container">
+          <img class="icon" src="images/program/forum.png" alt="image" />
```
Comment (Abdessittir) [inline index.html:115]: - [OPTIONAL] I highly recommend that you put a more descriptive `alt` to tell what this image is about, what do you think?

File: index.html [hunk 10]
```diff
@@ -0,0 +1,198 @@
...
+          <h3 class="h3">Forum</h3>
+          <p class="p">
+            Have the time to share your thoughts and opinions with experts for
+            each topic.
+          </p>
+        </div>
+
+        <!-- Box 4 -->
+        <div class="inner-container">
+          <img class="icon" src="images/program/workshop.png" alt="image" />
```
Comment (Abdessittir) [inline index.html:125]: - [OPTIONAL] I highly recommend that you put a more descriptive `alt` to tell what this image is about, what do you think?

File: index.html [hunk 11]
```diff
@@ -0,0 +1,198 @@
...
+          <h3 class="h3">Workshop</h3>
+          <p class="p">
+            Try creating your own work using open source license rather than
+            just watching at it.
+          </p>
+        </div>
+
+        <!-- Box 5 -->
+        <div class="inner-container">
+          <img class="icon" src="images/program/ccignite.png" alt="image" />
```
Comment (Abdessittir) [inline index.html:135]: - [OPTIONAL] I highly recommend that you put a more descriptive `alt` to tell what this image is about, what do you think?

File: index.html [hunk 12]
```diff
@@ -0,0 +1,198 @@
...
+        <h2 class="h2">Partner</h2>
+        <hr class="red-line" />
+      </div>
+
+      <div class="image-container">
+        <img class="image" src="images/partners/mozilla.png" alt="image" />
+        <img class="image" src="images/partners/netflix.png" alt="image" />
+        <img class="image" src="images/partners/airbnb.png" alt="image" />
+        <img class="image" src="images/partners/amazon.png" alt="image" />
+        <img class="image" src="images/partners/ibm.png" alt="image" />
```
Comment (Abdessittir) [inline index.html:174]: - [OPTIONAL] I highly recommend that you put a more descriptive `alt` to tell what this image is about, what do you think?

File: index.html [hunk 13]
```diff
@@ -0,0 +1,198 @@
...
+      </div>
+    </section>
+
+    <!-- Footer Section -->
+    <footer class="footer">
+      <img class="logo" src="images/logo.png" alt="image" />
```
Comment (Abdessittir) [inline index.html:180]: - [OPTIONAL] I highly recommend that you put a more descriptive `alt` to tell what this image is about, what do you think?

**Review by Abdessittir — APPROVED** [review]
Hi @peter-akhigbe , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



## Highlights
✅ No linter errors
✅ Good commit messages
✅ Stunning design 💯 
## Optional suggestions
- [ OPTIONAL] Check the comments below

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**